website/content/docs/commands/var/init.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: 'Commands: var init'
+description: |
+  Generate an example variable specification.
+---
+
+# Command: var init
+
+The `var init` creates an example variable specification file that can be used
+as a starting point to customize further.
+
+## Usage
+
+```plaintext
+nomad var init <filename>
+```
+
+When no filename is supplied, a default filename of "spec.nsv.hcl" or
```

Comment:
Should these still be "nsv" after the name change?

Reply:
Good catch! Fixed and fixed over here too: https://github.com/hashicorp/nomad/pull/14552